bootstrap.yml

```
@@ -0,0 +1,140 @@
+      echo "Unique userCapabilities names: \"$capabilityNames\""
+
+      # Prepare the parameters
+      parametersJson=$(jq -n --arg appComponents "- $capabilityNames" '{templateParameters: {appComponents: $appComponents}}')
+
+      echo "Parameters: \"$parametersJson\""
+      echo "Branch for PR: \"$branch\""
+
+      # Define the request body
+      bodyJson=$(jq -n --argjson parameters "$parametersJson" --arg branch "$branch" '{
+        resources: {
+          repositories:               
```

yamllint

[error] trailing spaces

(trailing-spaces)